packages/server/tsconfig.esm.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": "../../tsconfig.esm.json",
+    "compilerOptions": {
+        "outDir": "./dist/esm" /* Redirect output structure to the directory. */,
+        "rootDir": "./src" /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */,
+        "baseUrl": "."
```

Comment:
How is this config file compiling without the path mappings like in the cjs version